<launch>

     <node pkg="topology_mapping" type="topology_gap_analysis_node" name="topology_gap_analysis" output="screen"> 
     
     </node> 

     <!--node pkg="topology_mapping" type="topology_mapping_node" name="topology_mapping" output="screen"> 
     
     </node--> 


</launch>
